common/djangoapps/enrollment/serializers.py

```diff
@@ -99,3 +103,45 @@ class ModeSerializer(serializers.Serializer):
     description = serializers.CharField()
     sku = serializers.CharField()
     bulk_sku = serializers.CharField()
+
+
+# Edraak: adding custom serializer for enrollments
+class EdraakCourseEnrollmentSerializer(CourseEnrollmentSerializer):
+    edraak_course_details = EdraakCourseSerializer(source="course_overview")
+    is_certificate_allowed = serializers.SerializerMethodField()
+    specialization_slug = serializers.SerializerMethodField()
+
+    def get_is_certificate_allowed(self, obj):
+        from edraak_certificates.utils import is_certificate_allowed
```

Comment:
Why the local import?

Reply:
Because of these two errors: http://build.edraak.cc/blue/organizations/jenkins/edraak-platform/detail/PR-98/1/tests
Enrollment test are skipped for cms. But `edraak_certificates.utils` is importing stuff from `lms` that are not defined in `cms`, that causes the test environment to fail before getting to the skip part. So, I have to define extra unused environment variables inside cms envs, or just import it locally like this

what do you think @OmarEdraak ?

Reply:
Makes sense. Please document that in the code, at least a hint about it.

Reply:
Done